src/index/txindex.cpp

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <chainparams.h>
+#include <index/txindex.h>
+#include <init.h>
+#include <tinyformat.h>
+#include <ui_interface.h>
+#include <util.h>
+#include <validation.h>
+#include <warnings.h>
+
+static constexpr int64_t SYNC_LOG_INTERVAL = 30; // seconds
```

Comment:
remove `static`, that's the opposite of what you want (it reserves `.bss` space)